esmvalcore/preprocessor/_regrid.py

```diff
@@ -383,6 +383,18 @@ def extract_location(cube, location, scheme):
                          scheme)
 
 
+def _check_grid_discontiguities(cube, scheme):
+    """Check if there are grid discontiguities and set use_src_mask to True."""
+    try:
+        discontiguities = iris.util.find_discontiguities(cube)
+    except NotImplementedError:
```

Comment:
can the `NotImplementedError` be tossed by iris based on its version (if so, can you pls add a comment stating starting what version of iris that function is implemented)?

Reply:
Okey, I will browse through iris to see when that was added

Reply:
thinking about it even more (yeh Monday mornings) it'd be nice to have a debug warning about it too, if you find the iris version then that'd be good be included in the warning message :+1:

Reply:
We have pinned iris to the latest version, so this should be needed.